vault/resource_rabbitmq_secret_backend_role.go

```diff
@@ -168,12 +168,19 @@ func rabbitMQSecretBackendRoleRead(d *schema.ResourceData, meta interface{}) err
 	d.Set("backend", strings.Join(pathPieces[:len(pathPieces)-2], "/"))
 	d.Set("name", pathPieces[len(pathPieces)-1])
 
-	if err := d.Set("vhost", flattenRabbitMQSecretBackendRoleVhost(secret.Data["vhosts"].(map[string]interface{}))); err != nil {
-		return fmt.Errorf("Error setting vhosts in state: %w", err)
+	vhosts := secret.Data["vhosts"]
+	vhostTopics := secret.Data["vhost_topics"]
+
+	if vhosts != nil {
+		if err := d.Set("vhost", flattenRabbitMQSecretBackendRoleVhost(secret.Data["vhosts"].(map[string]interface{}))); err != nil {
```

Comment:
Looks like we don't use `vhosts` from above.

Reply:
Not sure, but I feel like that we can inline the variable assignment like:
```
	if vhosts, ok := secret.Data["vhosts"]; ok && vhosts != nil {
		if err := d.Set("vhost", flattenRabbitMQSecretBackendRoleVhost(vhosts.(map[string]interface{}))); err != nil {
			return fmt.Errorf("error setting vhosts in state: %w", err)
		}
	}
```

Same pattern could work for `vhost_topic` below.

Reply:
Fixed in bdf259f